### ISSUE TITLE
Release new gem version for Omniauth v2 support

### ISSUE THREAD

**Issue by olbrich** [isaacsanders/omniauth-stripe-connect#67]
It seems like omniauth support has been merged for a while now (see #65 and #64). It would be good to release a new version of the gem so that other gems and projects that depend on this can update to newer versions of omniauth.

This will allow others to mitigate [this CVE for the omniauth gem](https://nvd.nist.gov/vuln/detail/CVE-2015-9284)

**Comment by johnnyshields** [comment]
@isaacsanders can you please help with this? This is needed for a CVE fix please.

**Comment by fidalgo** [comment]
@isaacsanders Is there anything I can help?

**Comment by jftobias** [comment]
@isaacsanders Hey dude! Thanks for your valuable contribution. Is there any scheduled day for the release? Thanks

**Comment by catsarebetter98** [comment]
@isaacsanders Would love to see this released, it would help out a lot with our applications :)

**Comment by sbounmy** [comment]
@isaacsanders if you are no longer active, maybe you could add new contributors to maintain ?

**Comment by Fire-Dragon-DoL** [comment]
Agree, I'm in need of the updated code, for now I'm using the github reference, but it would be nice if this was released officially

**Comment by mmenisy** [comment]
It seems like this is no longer maintained. For anyone looking for a quick solution, just copy the strategy from here and use it in your own project right away.
